src/components/common/InvitationBlock/InvitationBlock.tsx

```diff
@@ -23,30 +24,43 @@ const MSG = defineMessages({
   },
   inviteBlockDescription: {
     id: `${displayName}.inviteBlockDescription`,
-    defaultMessage:
-      'You can invite only one member to create a colony of their own using the new app during the private beta with this custom invite link: app.colony.io/createcolony/{invitationCode}',
+    /* eslint-disable max-len */
+    defaultMessage: `{showDescription, select,
+        true {You can invite only one member to create a colony of their own using the new app during the private beta with this custom invite link: }
+        other {}
+      }{inviteLink}`,
+    /* eslint-enable max-len */
   },
 });
 
-const InvitationBlock = () => {
+interface Props {
+  showDescription?: boolean;
+}
+
+const InvitationBlock = ({ showDescription = true }: Props) => {
   const { user } = useAppContext();
   const invitationCode = user?.privateBetaInviteCode?.id;
-  const inviteLink = `app.colony.io/createcolony/${invitationCode}`;
+  const inviteLink = `app.colony.io/create-colony/${invitationCode}`;
```

Comment:
Why is the url hardcoded ? And why is this not a hook that just generates your invite code ?

Reply:
Added a new hook, let me know if something like this is what you had in mind 👍